test/Microsoft.AspNetCore.Mvc.FunctionalTests/RazorPagesWithBasePathTest.cs

```diff
@@ -323,7 +323,7 @@ public async Task PagesInAreas_CanGenerateLinksToControllersAndPages()
             var expected =
 @"<a href=""/Accounts/Manage/RenderPartials"">Link inside area</a>
 <a href=""/Products/List/old/20"">Link to external area</a>
-<a href=""/Accounts"">Link to area action</a>
+<a href="""">Link to area action</a>
```

Comment:
I think it would be good to change to the test code to be successful, or even better skip the test and link to the issue.